### ISSUE TITLE
AssertionError during import

### ISSUE THREAD

**Issue by alex-d13** [theislab/sfaira#353]
**Describe the bug**
When I try to load sfaira with `import sfaira`, I get the following error:
```
>>> import sfaira
Downloading: cellosaurus.obo
WARNING:root:name and ontology keys are both missing
Downloading: v2021-02-01_cl.obo
Downloading: hsapdv.obo
Downloading: mmusdv.obo
Downloading: mondo.obo
Ontology <class 'sfaira.versions.metadata.base.OntologyMondo'> is not a DAG, treat child-parent reasoning with care.
Downloading: uberon.obo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.9/site-packages/sfaira/__init__.py", line 4, in <module>
    import sfaira.consts
  File "/usr/local/lib/python3.9/site-packages/sfaira/consts/__init__.py", line 5, in <module>
    OCS = OntologyContainerSfaira()
  File "/usr/local/lib/python3.9/site-packages/sfaira/consts/ontologies.py", line 35, in __init__
    self.organ = OntologyUberon()
  File "/usr/local/lib/python3.9/site-packages/sfaira/versions/metadata/base.py", line 731, in __init__
    assert x[2] in edge_types, x
AssertionError: ('CL:0000015', 'GO:0007283', 'capable_of')
```

**To Reproduce**
I used a Docker image with the following Dockerfile:

``` 
FROM python:3

RUN pip install --no-cache-dir --upgrade pip && \
    pip install --no-cache-dir sfaira
```
Then I ran `import sfaira` inside this Docker container.

**Expected behavior**
I would expect to load the package without an error message.

**System [please complete the following information]:**
 - OS: Ubuntu 18.04.5 LTS
 - Language Version: Python 3.9.7
 - Virtual environment: -

**Comment by davidsebfischer** [comment]
That looks like an update to UBERON, we are working on getting versioned UBERON downloads which will mitigate this in the future (https://github.com/theislab/sfaira/issues/340). In the meantime, I will address this error in particular by updating our data library, this will be done next few days!

**Comment by davidsebfischer** [comment]
UBERON versioning is done and merged, I think your issue was actually a different one that was already solved on dev, I just released sfaira 0.3.5 so this fix should now also reflect in the pypi installation in your docker! Sfaira is still changing a lot, so I would recommend that you fix the sfaira version in your docker (to 0.3.5, the newest one), so that you can easily tell later which one you have.
